command/bridge/deploy/deploy.go

```diff
@@ -15,7 +15,7 @@ import (
 	"github.com/0xPolygon/polygon-edge/command"
 	"github.com/0xPolygon/polygon-edge/command/bridge/helper"
 	cmdHelper "github.com/0xPolygon/polygon-edge/command/helper"
-	"github.com/0xPolygon/polygon-edge/consensus/polybft"
+	polycfg "github.com/0xPolygon/polygon-edge/consensus/polybft/config"
```

Comment:
```suggestion
	polyCfg "github.com/0xPolygon/polygon-edge/consensus/polybft/config"
```

Reply:
From Golang [guidelines](https://go.dev/blog/package-names) on package name:

`Good package names are short and clear. They are lower case, with no under_scores or mixedCaps`.